usaspending_api/common/serializers.py

```diff
@@ -19,56 +19,64 @@ def __init__(self, *args, **kwargs):
             for field in children.keys():
                 child_args = {
                     **children[field].get("kwargs", {}),
-                    "context": {**self.context, "child": True},  # The child tag should be removed when child field limiting is implemented
-                }
+                    "context": {**self.context, "child": True}
+                }  # The child tag should be removed when child field limiting is implemented
                 self.fields[field] = children[field]["class"](**child_args)
         except AttributeError:
             # We don't have any nested serializers
             pass
 
         request = self.context.get('request')
-        if request and not self.context.get("child", False):  # The child check should be removed when child field limiting is implemented
-            # workaround to use this serializer with both GET and POST
+        if request:
```

Comment:
Removing the child check here - does the endpoint still behave properly when a filtered field is that of a child? i.e.:
```
"filters": [
{
      "field": "procurement__type",
      "operation": "is_null",
      "value": false
}
]
```

Without this check before, it would return the summary award X times, where X is the number of procurements. Your other changes may account for this but I want to make sure it doesn't creep back in.